plugins/metrics/src/org/apache/cloudstack/response/ClusterMetricsResponse.java

```diff
@@ -128,10 +128,10 @@ public void setCpuAllocated(final Long cpuAllocated, final Long cpuTotal) {
         }
     }
 
-    public void setCpuMaxDeviation(final Double maxCpuDeviation, final Double totalCpuUsed, final Long totalHosts) {
-        if (maxCpuDeviation != null && totalCpuUsed != null && totalHosts != null && totalHosts != 0) {
-            final Double averageCpuUsage = totalCpuUsed / totalHosts;
-            this.cpuMaxDeviation = String.format("%.2f%%", (maxCpuDeviation - averageCpuUsage) * 100.0 / averageCpuUsage);
+    public void setCpuMaxDeviation(final Double maxCpuUsagePercentage, final Double totalCpuUsedPercentage, final Long totalHosts) {
+        if (maxCpuUsagePercentage != null && totalCpuUsedPercentage != null && totalHosts != null && totalHosts != 0) {
+            final Double averageCpuUsagePercentage = totalCpuUsedPercentage / totalHosts;
+            this.cpuMaxDeviation = String.format("%.2f%%", (maxCpuUsagePercentage - averageCpuUsagePercentage) * 1.0 / averageCpuUsagePercentage);
```

Comment:
is the '* 1.0' needed when it is devided next? I would rather see a cast if it is, in this case